scm-record/src/ui.rs

```diff
@@ -124,13 +124,18 @@ pub enum Event {
     PageUp,
     PageDown,
     FocusPrev,
-    FocusPrevSameKind, // focus on the previous item of the same kind (i.e. file, section, line)
+    // Move focus to the previous item of the same kind (i.e. file, section, line).
+    FocusPrevSameKind,
     FocusPrevPage,
     FocusNext,
-    FocusNextSameKind, // focus on the next item of the same kind
+    // Move focus to the next item of the same kind.
+    FocusNextSameKind,
     FocusNextPage,
     FocusInner,
-    FocusOuter,
+    // If `fold_section` is true, and the current section is expanded, the
+    // section should be collapsed without moving focus. Otherwise, move the
+    // focus outwards.
+    FocusOuter { fold_section: bool },
```

Comment:
nit: Let's make these doc-comments so that they appear in the IDE upon hover, etc.? (The previous versions of these comments probably should also have been doc-comments.)

Reply:
Good idea. Done (locally, I'll push the change after resolving the other comments).